### DISCUSSION TITLE
Expose `ElmHtml.ToString.FormatOptions` in `Api.routes`

### DISCUSSION THREAD

**Discussion by klaftertief** [dillonkearns/elm-pages#304] [Ideas]
The `Api.routes` function to create arbitrary static files is automatically called with the `htmlToString : Html Never -> String)` function. It is currently configured to render without indentation or newlines.
The underlying `ElmHtml.ToString` module supports format options to configure the rendering. Would it be possible to extend the `Api.routes` API in an upcoming major release to be able to use the `ElmHtml.ToString.FormatOptions`?

The reason is because I'd like to generate files that are meant to be user-readable and diff-able.

**Comment by dillonkearns** [comment]
That seems like a good idea, I went ahead and added it as a parameter here, it will be included in the 3.0 release 👍 

https://github.com/dillonkearns/elm-pages-v3-beta/commit/6a067671200f896c2a9b10a31aab307f88943973#diff-72c6a3463bd479a0c95cbe37b7e94df21ca45fd2b7f8e3cf68241681aff1cf18

**Reply by klaftertief** [reply]
Awesome, thank you!

**Reply by dillonkearns** [reply]
My pleasure, thanks for the feedback!
